akka-bbb-apps/src/main/scala/org/bigbluebutton/core/BigBlueButtonActor.scala

```diff
@@ -114,7 +120,9 @@ class BigBlueButtonActor(
       case None =>
         log.info("Create meeting request. meetingId={}", msg.body.props.meetingProp.intId)
 
-        val m = RunningMeeting(msg.body.props, outGW, eventBus)
+        val clientConfigForMeeting: Map[String, Object] = clientConfigurationFromFile
+
+        val m = RunningMeeting(msg.body.props, outGW, eventBus, clientConfigForMeeting)

```

Comment:
I think it's better to load the config Map on the constructor of the `RunningMeeting` instead of passing it as a parameter.